x/programs/program/context.go

```diff
@@ -0,0 +1,11 @@
+package program
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+type Context struct {
+	ProgramID        ids.ID   `json:"program"`
+	Actor            [32]byte `json:"actor"`
+	OriginatingActor [32]byte `json:"originating_actor"`
```

Comment:
I don't think we need to change this now, but I like @patrick-ogrady's idea of a stack of actors... I can already see myself responding to issues saying, "I think you meant to use `actor`, not `originating_actor`"